src/ray/protobuf/serve.proto

```diff
@@ -88,6 +88,8 @@ message DeploymentConfig {
   AutoscalingConfig autoscaling_config = 10;
 
   string version = 11;
+
+  repeated string user_configured_options = 12;
```

Comment:
why is this `repeated string` type?

Reply:
In the protobuf, `user_configured_options` is a list of strings. Each string is a deployment options that was manually tuned by the user.

I used `repeated string` because I felt it best matched a list of strings. Is there another datatype I should use?

Reply:
> Each string is a deployment options that was manually tuned by the user.

Ahhh these are options **keys**. not the value. can you change the variable name so it implies that?

Reply:
Good idea, I renamed `user_configured_options` to `user_configured_option_names`.